src/main/java/com/company/M11/Main.java

```diff
@@ -0,0 +1,76 @@
+package com.company.M11;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+public class Main {
+    public static void main(String[] args) {
+//        task1();
+//        task2();
+//        task3();
+//        task4();
+//        task5();
+    }
+
+    private static void task1() {
+        List<String> names = Arrays.asList("John", "Bill", "Anthon", "Alex");
+
+        IntStream.range(0, names.size())
+                .filter(index -> index % 2 != 0)
+                .forEachOrdered(index -> System.out.println(index + ". " + names.get(index)));
+    }
+
+    private static void task2() {
+        List<String> names = Arrays.asList("John", "Bill", "Anthon", "Alex");
+        names.stream()
+                .sorted()
+                .map(String::toUpperCase)
+                .forEachOrdered(System.out::println);
+    }
+
+    private static void task3() {
+        String[] inputArray = {"1, 2, 0", "4, 5"};
+        int[] numbers = Arrays.stream(inputArray)
+                .flatMapToInt(s -> Arrays.stream(s.split(", "))
+                        .mapToInt(Integer::parseInt))
+                .toArray();
+        Arrays.stream(numbers).
+                sorted()
+                .forEach(number -> System.out.print(number + ", "));
```

Comment:
good job! 
- and let's try to combine this two stream in one
- also let's return a result as string value

Reply:
fixed